modules/nw-multus-tap-object.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: REFERENCE
+[id="nw-multus-tap-object_{context}"]
+= Configuration for a TAP additional network
+
+The following object describes the configuration parameters for the TAP CNI
+plugin:
+
+.TAP CNI plugin JSON configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
+
+|`name`
+|`string`
+|The value for the `name` parameter you provided previously for the CNO configuration.
+
+|`type`
+|`string`
+|The name of the CNI plugin to configure: `tap`.
+
+|`mac`
+|`string`
+|Optional: Request the specified MAC address for the interface.
+
+|`mtu`
+|`integer`
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+
+|`selinuxcontext`
+|`string`
+|Optional: The SELinux context to associate with the tap device.
```

Comment:
I'm not sure if it's optional. Without correct SELinux context pod will not be able to use connected tap device. @mlguerrero12 Please advice.

Reply:
Yes maybe in the context of OpenShift it is not optional I was going with https://www.cni.dev/plugins/current/main/tap/ but AFAIK OCP is a system with "with the selinux security module enabled"

Reply:
Nikita is right, strictly speaking this parameter is optional, but for us (OCP)i is not. This documentation is focus in OCP, so we should somehow make this clear to the users.

Reply:
@mlguerrero12 should the vale in the context of OCP always be` system_u:system_r:container_t:s0`

Reply:
@mlguerrero12 @kononovn I added that clarification with the additional line

Reply:
looks good. We should also add here for the tap plugin that the SELinux boolean container_use_devices must enabled on the host (through a Machine Config) for the tap device to be created when the container_t SELinux context is used.

Reply:
I did add here, but I have it also in https://github.com/openshift/openshift-docs/pull/60840 specifically this section https://60840--docspreview.netlify.app/openshift-enterprise/latest/networking/hardware_networks/using-dpdk-and-rdma.html#nw-running-dpdk-rootless-tap_using-dpdk-and-rdma. What do you think, is it OK to leave here in this PR or is there too much info in that note ?

Reply:
Hi @mlguerrero12 I moved that additional clarification to under the example and I will add cross refs in a separate PR when all these kinda related PRs are merged (facilitating rootless DPDK). I hope that is OK.